README.md

```diff
@@ -79,24 +79,10 @@ Autoloading is PSR-0 friendly.
 
 ### Via [Packagist](https://packagist.org/packages/ronanguilloux/isocodes) & [Composer](http://getcomposer.org/doc/00-intro.md)
 
-Create a composer.json file:
-
-```json
-{
-    "require": {"ronanguilloux/isocodes": "@stable"}
-}
-```
-
-Grab composer:
-
-```bash
-$ curl -s http://getcomposer.org/installer | php
-```
-
-Run install (will build the autoloading):
+Require the latest version of `ronanguilloux/isocodes` with Composer
 
 ```bash
-$ php composer.phar install
+composer require ronanguilloux/isocodes
 ```
 
```

Comment:
`@stable` is just a flag to change the stability per package. Usually, the [minimum-stability](https://getcomposer.org/doc/04-schema.md#minimum-stability) is already set to `stable`. But if it's `dev` you can force a package to use `@stable` (or vice versa), see https://getcomposer.org/doc/04-schema.md#package-links

But this doesn't actually enforce a version. So people requiring `@stable` will currently get `1.1.4` when they update. But when you release 2.0, they will get that instead. That would probably be a breaking change.

If you follow [SemVer](http://semver.org/) (which you should), you should encourage `~1.1` instead. If you don't follow SemVer and 1.2 could be breaking, you schould encourage `1.1.x`.